### DISCUSSION TITLE
I wrote "pip install py-solc-x" in terminal, but it throws error

### DISCUSSION THREAD

**Discussion by Forealthy** [smartcontractkit/full-blockchain-solidity-course-py#1597] [Q&A] [answered]
I want to install "py-solc-x", and have written "pip install py-solc-x" in the terminal, but it didn't work, and throw errors.

This is order I wrote in the terminal:
![pip_py-solc-x0](https://user-images.githubusercontent.com/109839460/180632550-b7d00b8d-8f8e-499a-9850-017691cabc3f.png)
This is error:
![pip_py-solc-x](https://user-images.githubusercontent.com/109839460/180632498-0a04cc1a-39ba-43c4-99a3-4cddb175d6d3.png)
......tons of error like these......
![pip_py-solc-x2](https://user-images.githubusercontent.com/109839460/180632504-0c7c1271-49be-4d38-b0ec-11fe3e29c602.png)

Can anyone help?

**Comment by cromewar** [comment, marked as answer]
Hello @Forealthy 
Please do not use conda/anaconda environments for web3 libraries, they have a bunch of conflicts and problems, Instead use  a regular python installation with a virtualenv, this should solve your problem.

**Reply by Forealthy** [reply]
Thanks a lot!
